modules/runner/README.md

```diff
@@ -203,3 +203,10 @@ Besides local node stopping logic two more actions took place on a cluster as a
  * Both range and watch cursors will be removed on server side. Given process is linearized with meta storage
   operations by using a meta storage raft.
  * Baseline update and corresponding baseline recalculation with ongoing partition raft groups redeployment.
+
+## Threading
```

Comment:
This parts looks like out of nowhere, are you sure this is the correct place or that it even needs to be mentioned?

Reply:
The section actually looks alien here, I moved it to the `threading.md`